common/spring-cloud-stream-modules-common-configuration/src/main/java/org/springframework/cloud/stream/module/tcp/AbstractTcpConnectionFactoryProperties.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Interesting; this file started life as you `AbstractTcpConnectionFactoryOptionsMetadata` in XD; hence your author credit :smile: